docs/managing-e2e-tests.md

```diff
@@ -115,39 +109,16 @@ Following the `1.15` example from earlier, a job called `ci-kubernetes-e2e-kubea
 
 Removed:
 ```
-- name: kubeadm-gce-1.12
-  test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-12
+- name: kubeadm-kind-1-12
+  test_group_name: ci-kubernetes-e2e-kubeadm-kind-1-12
 ```
 
 Added:
 ```
-- name: kubeadm-gce-1.15
-  test_group_name: ci-kubernetes-e2e-kubeadm-gce-1-15
+- name: kubeadm-kind-1-15
+  test_group_name: ci-kubernetes-e2e-kubeadm-kind-1-15
```

Comment:
If possible also add a not about making nature that run frequency for old release is 12h, master 2h

Reply:
adding a section `Job run frequency`